spec/spec_helper.rb

```diff
@@ -150,3 +150,11 @@ def conjur_server_dir
   # Navigate from its directory (/bin) to the root Conjur server directory
   Pathname.new(File.join(File.dirname(conjurctl_path), '..')).cleanpath
 end
+
+# Toggle a feature flag on for a particular set of tests.
+def with_feature_enabled(feature, &block)
```

Comment:
Handling the flag toggle with dependency injection means this can be removed.  It's MUCH cleaner and prevents us from needing to mutate the environment variable during testing.  I'm removing this and the gem.